hack/local-setup.sh

```diff
@@ -50,6 +49,11 @@ deployCertManager() {
   ${KUSTOMIZE_BIN} build ${CERT_MANAGER_KUSTOMIZATION_DIR} --enable-helm --helm-command ${HELM_BIN} | kubectl apply -f -
   echo "Waiting for Cert Manager deployments to be ready..."
   kubectl -n cert-manager wait --timeout=300s --for=condition=Available deployments --all
+
+  kubectl delete validatingWebhookConfiguration mctc-cert-manager-webhook
+  kubectl delete mutatingWebhookConfiguration mctc-cert-manager-webhook
```

Comment:
Why are we doing this? If they aren't required, can they be prevented from being created in the first place via some helm chart config option?

Reply:
They don't work with self signed certs so locally they kept giving errors

Reply:
I've seen this used before to filter resources from the output of helm & kustomize
```
kustomize build . | docker run --rm -i ryane/kfilt -i kind=validatingWebhookConfiguration | kubectl apply -f -
```

You can put it in a var for easier use as well
```
KFILT = docker run --rm -i ryane/kfilt
kustomize build . | $(KFILT) -i kind=validatingWebhookConfiguration | kubectl apply -f -
```